src/jrd/trace/TraceManager.h

```diff
@@ -53,7 +44,7 @@ class jrd_tra;
 class DsqlRequest;
 class Service;
 
-class TraceManager
+class TraceManager : public ServerTraceManager
```

Comment:
It is very confused.
I suppose the base class should be a TraceManager, and derived classes should be a JrdTraceManager,  (Net)ServerTraceManager, etc